js/calendar.js

```diff
@@ -147,6 +161,41 @@ const goToMonth = (direction) => {
   renderCalendarView(viewYear, viewMonth); // 달력 다시 렌더링
 };
 
+// 선택한 날짜에 맞게 Weekly 구현
+const renderWeeklyView = (e) => {
+  // 선택한 요소(달력)에서 태그를 가져와 weekly의 date-container에 추가
+
+  // 1. 선택한 요소는 달력에서 클릭했으므로 date-container가 존재
+  const $selectedDateBox = e.target;
+  const $selectedCalendar = $selectedDateBox.parentElement;
+
+  // 1-1. data-date-idx 는 언제나 달력에서 0 ~ 41까지 고정됨
+  // 1-2. data-date-idx 를 7로 나눈 나머지가 해당 날짜의 요일
+  const selectedDateIdx = $selectedDateBox.firstElementChild.dataset.dateIdx;
+  const selectedDay = selectedDateIdx % 7;
+
+  // 2. 선택한 날짜가 있는 주의 첫번째 날짜(일요일)의 data-date-idx 추출
+  const startDateIdx = selectedDateIdx - selectedDay;
+  
+  // console.log($selectedCalendar.children[startDateIdx]);
+  // console.log($selectedCalendar.children[startDateIdx+1]);
+
+  // 3. weekly의 date-container에 추가
+  
+  const tagDatesWeek = [];
+
+  for (let i = startDateIdx; i < 7; i++) {
+    // 3-1. weekly의 시작날짜 태그부터 마지막(토요일) 태그를 배열에 추가
+    tagDatesWeek.push($selectedCalendar.children[i].outerHTML);
+  }
+
```

Comment:
둘 째주 부터 출력되려면 이렇게 되어야 할 것 같아요
```suggestion
  for (let i = startDateIdx; i < 7 + startDateIdx; i++) {
    // 3-1. weekly의 시작날짜 태그부터 마지막(토요일) 태그를 배열에 추가
    tagDatesWeek.push($selectedCalendar.children[i].outerHTML);
  }

```

Reply:
아 맞아요 수정하겠습니다!